api/v1/tests.py

```diff
@@ -7,25 +7,32 @@
 from fixtureless.factory import create
 
 from recipes.models import Recipe
+from recipes.serializers import RecipeSerializer
 
 
 class TestRecipeApi(APITestCase):
     def setUp(self):
         self.user = get_user_model().objects.create(username='test_user')
         self.recipe = create(Recipe, {'title': 'testing'})
+        self.recipe_list_url = reverse_lazy('api:v1:recipe-list')
+        self.recipe_detail_url = reverse_lazy('api:v1:recipe-detail', kwargs={'pk': self.recipe.pk})
 
     def test_recipe_list(self):
         """
         Verify GET returns Recipe list
         """
-        response = self.client.get(reverse_lazy('api:v1:recipe-list'))
+        response = self.client.get(self.recipe_list_url)
+        recipes = Recipe.objects.all()
+        serializer = RecipeSerializer(recipes, many=True)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Recipe.objects.count(), 1)
+        self.assertEqual(Recipe.objects.get().title, 'testing')
```

Comment:
if the test setup ever changes to add more recipes during setup time, this would fail. Instead of doing an empty `get()` call, we can specify the pk of the specific recipe you want. `get(pk=self.recipe.pk)`

Reply:
you can also chain this from the cached variable `recipes`, just fyi. `recipes.get(pk=self.recipe.pk)`